charts/k8s-monitoring/ci/ci-values.yaml

```diff
@@ -49,8 +53,15 @@ test:
   attempts: 20
   extraQueries:
     # Check that the external label is properly being set
-    - query: "kube_node_info{cluster=\"ci-test-cluster\", color=\"red\"}"
+    - query: "count(kube_node_info{cluster=\"ci-test-cluster\", color=\"red\"})"
       type: promql
+      expect:
+        value: 1
```

Comment:
This only runs on the test cluster right? So we always know there will be 1 node?

Reply:
Yeah, this is the values run during CI.